testcontainers/rabbitmq.py

```diff
@@ -0,0 +1,76 @@
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+from .core.container import DockerContainer
+from .core.waiting_utils import wait_container_is_ready
+import os
+import pika
+
+
+class RabbitmqContainer(DockerContainer):
+    RABBITMQ_DEFAULT_USER = os.environ.get("RABBITMQ_DEFAULT_USER", "guest")
+    RABBITMQ_DEFAULT_PASS = os.environ.get("RABBITMQ_DEFAULT_PASS", "guest")
+    SERVICE_STARTED = r"Server startup complete;"
+
+    def __init__(self, image="rabbitmq:latest"):
```

Comment:
`rabbitmq:latest` does not have `rabbitmqadmin`, so all methods below which are using it do not work